cosmoz-bottom-bar.js

```diff
@@ -184,27 +199,27 @@
 			if (matchParent) {
 				return this.parentElement;
 			}
-			return null;
```

Comment:
`computeMatchElementHeight` can not be `undefined` because then dependency property `_computedBarHeight` will not be computed. So this function must return `null`.